source/style-guide/style/tables.txt

```diff
@@ -49,8 +49,8 @@ within tasks, procedures, and tutorials don't require titles.
 
 When creating table titles, use the following guidelines:
 
-- Use sentence-style capitalization for table titles. However, for
-  words that are always uppercase or always lowercase, match that case.
+- Use headline-style capitalization for table titles. See 
+  :ref:`titles-and-headings` for a list of capitalization guidelines.
```

Comment:
[nit, suggestion]

Our [style guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear) suggests putting the purpose of the link first, followed by the target.

Suggest modifying to:

"For a list of capitalization guidelines, see (link)"